app/components/post_profile_picture/index.js

```diff
@@ -19,14 +20,20 @@ function mapStateToProps(state, ownProps) {
     const post = ownProps.post;
     const user = getUser(state, post.user_id);
 
+    // if it's relative we assume it's a link to an emoji
+    const isEmoji = post?.props?.override_icon_url?.startsWith('/') || false; // eslint-disable-line camelcase
```

Comment:
I don't know if we can make this assumption since it's definitely possible that an integration could use a relative link for its profile image, particularly if that webhook is being used by a plugin that hosts the image itself.

Maybe when we populate the `override_icon_url`, we could just leave the `override_icon_emoji` prop on the post and use that to know for certain that this is an emoji? I think it's totally fine to have both fields since that supports both new and old apps

Reply:
Agreed. Changed.